.github/workflows/CI.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        eslint: [8.x, "next"]
+        eslint: [8.x, "9.x"]
```

Comment:
🤔 Would it be worth leaving the `next` here? for the next eslint release?

Reply:
eslint doesn't release pre-versions as often as many other packages do, and based on history, after that it only happens when major releases are made, ~1 year. We can add that when eslint v10 goes to pre-releases. :)